Project/backend/codebase/graphCreator/pdfHandler.py

```diff
@@ -0,0 +1,36 @@
+import os
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from langchain_community.document_loaders import PyPDFLoader
+
+
+def proccess(filename):
```

Comment:
I would give a more descriptive name to function since `proccess` is kinde general. Maybe something along `process_pdf_into_chunks` would be better